### PR TITLE
Groom changelog for 1.8.0 stable release

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What
Groomed changelog

# Why
1.8.0 stable release

# How Tested

# Process & policy checklist

- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.stable.md [hunk 1]
```diff
@@ -4,103 +4,44 @@ This log was last generated on Thu, 07 Sep 2023 02:42:30 GMT and should not be m
...
 <!-- Start content -->
 
-## [1.8.0](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.8.0)
+## [1.8.0](https://github.com/azure/communication-ui-library/tree/1.8.0)
 
 Thu, 07 Sep 2023 02:42:30 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.8.0-beta.1...@azure/communication-react_v1.8.0)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.8.0-beta.1...1.8.0)
+
+### Capabilities - General Availability
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.stable.md:12]: Did raise hand and background effects also hit GA in this release?

Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.stable.md:12]: Not sure if it is on feature driver or release driver to create these changelog entries...
Maybe release driver asks feature teams for a snippet to paste here

**Review by vhuseinova-msft — APPROVED** [review]

File: packages/communication-react/CHANGELOG.stable.md (hunk 1 above)
Comment (mgamis-msft) [reply packages/communication-react/CHANGELOG.stable.md:12]: I see. I was not sure if we have to snippet for features that are in beta going to stable. But it makes sense to do so. I can just copy the snippet from the beta changelog for each feature being promoted to GA. Does that sound okay?

File: packages/communication-react/CHANGELOG.stable.md [hunk 2]
```diff
@@ -4,103 +4,74 @@ This log was last generated on Thu, 07 Sep 2023 02:42:30 GMT and should not be m
...
+
+### Optimal Video Count - General Availability
+The Web UI Calling and CallWithChat composite now supports optimal video count which optimally control the maximum number of remote video renderers to ensure good video quality.
+
+### Closed Captions - General Availability
+The Web UI Calling and CallWithChat composite now supports Closed Captions in Teams Interop and Teams CTE scenarios.
+
+This feature includes:
+- Ability to enable, and show/hide closed captions in a call 
+- Choose or change the captions spoken language
```
Comment (carocao-msft) [inline packages/communication-react/CHANGELOG.stable.md:26]: Choose or change the captions spoken language and caption language

**Review by edwardlee-msft — APPROVED** [review]

**Comment by edwardlee-msft** [comment]
Nice, thanks for going back and fixing the links for the older changelogs

**Pull request merged**